<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimDialog</class>
 <widget class="QDialog" name="SimDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Entities</string>
  </property>
  <widget class="OGLWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>681</width>
     <height>681</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>700</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>705</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Press anywhere on the map to add entity</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myMain</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
